tests/protractor/utils.js

```diff
@@ -11,15 +11,24 @@ const _ = require('underscore'),
 let originalSettings;
 
 const request = (options, debug) => {
+  if (typeof options === 'string') {
+    options = {
+      path: options
+    };
+  }
+
   const deferred = protractor.promise.defer();
 
   options.hostname = constants.API_HOST;
   options.port = constants.API_PORT;
-  options.auth = auth.user + ':' + auth.pass;
+  options.auth = options.auth || auth.user + ':' + auth.pass;
 
   if (debug) {
-    console.log('REQUEST');
+    console.log('!!!!!!!REQUEST!!!!!!!');
+    console.log('!!!!!!!REQUEST!!!!!!!');
     console.log(JSON.stringify(options));
+    console.log('!!!!!!!REQUEST!!!!!!!');
+    console.log('!!!!!!!REQUEST!!!!!!!');
```

Comment:
I assume this is wanted, but just in case...

Reply:
Yeah it's intentional. We could standardise or something, but basically it was impossible to see requests in debug mode in output without it, because there is so much api spam. This only outputs if you explicitly add `true` to the end of a specific request in your tests.